public/app/percona/backup/components/BackupInventory/RestoreBackupModal/RestoreBackupModal.styles.ts

```diff
@@ -0,0 +1,49 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+export const getStyles = ({ colors, typography, spacing }: GrafanaTheme) => ({
+  label: css`
+    display: block;
+    text-align: left;
```

Comment:
isn't text alignment to the left the default?

Reply:
This style is being used on other places because we don't have any `Select` on core yet, so we use this when we want to mimic the core's style (e.g. `AddNotificationChannelModal`), so I've just copied it from there (see conversation [here](https://perconacorp.slack.com/archives/G017YQ1T155/p1615981974004500)).

Reply:
Can you be more specific? Where did you copy it from? If this code is here to stay, let's make it consistent with our standards anyway. What if we create our Select, I'm pretty sure we won't remember that this `label` style is related to the select component when we'll have to remove it :)

Reply:
It's both on `AddNotificationChannelModal.styles.ts` and `AddAlertRuleModal.styles.ts` (this one seems was even done by you)

Reply:
Check here: https://github.com/percona-platform/grafana/pull/26/files#diff-b6533a50f97fa4cf6925f00513bcf0e933c3fec0f922cf28811c47ccdbc49c4dR5

Reply:
Regarding the replacement later on: both the `Select` and the `label` are wrapped within a `Field` component, so replacing it should leave no label behind.
Whan we can do for now is to create a component that just renders that `Field` with the label and a slot for children, which might be `Select` or other thing

Reply:
My point is that if this code is getting duplicated like that, I'd prefer to have it in one place. That might be outside the scope of this PR though. Still, the fact that some code is already in our codebase doesn't mean it's perfect, we should always review it anyway :)

Again, if it's going to be removed soon, ignore this, especially considering it's a minor thing :)

Reply:
> Regarding the replacement later on: both the Select and the label are wrapped within a Field component, so replacing it should leave no label behind.

That's true for the React code, but not necessarily for CSS :) The linter wouldn't complain about unused variables in that case, since those are object elements.

Reply:
I agree that this should be centralised 👍  
Let's take care of that on other issue: https://jira.percona.com/browse/PMM-7922